src/main/scala/org/broadinstitute/hail/driver/LinearRegressionCommand.scala

```diff
@@ -34,6 +40,12 @@ object LinearRegressionCommand extends Command {
   def run(state: State, options: Options): State = {
     val vds = state.vds
 
+    if (options.minAC < 1)
+      fatal(s"Minumum alternate allele count must be a positive integer, got ${ options.minAC }")
+
+    if (options.minAF < 0d || options.minAF > 1d)
+      fatal(s"Minumum alternate allele frequency must be between 0.0 and 1.0, got ${ options.minAF }")
```

Comment:
Great.  I want to allow both to be specified, as I say in the docs that it is equivalent to giving the stricter of the two, and users may want to ensure both hold without doing a separate calculation ahead of time (also note that if there are missing phenos or covariates it isn't obvious until you run linreg how many samples will actually go into the regression).